src/main/java/by/mifort/automation/hr/dev/dto/CandidateDto.java

```diff
@@ -107,6 +113,10 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return ToStringBuilder.reflectionToString(this);
+        return "CandidateDto{" +
```

Comment:
Example
return new ToStringBuilder(this).
       append("name", name).
       append("age", age).
       append("smoker", smoker).
       toString();

Reply:
fixed missclicked